systemd/ocp-mco-sshkey.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -x
+
+source /usr/local/bin/crc-systemd-common.sh
+export KUBECONFIG="/opt/kubeconfig"
+
+pub_key_path="/opt/crc/id_rsa.pub"
+
+if [ ! -f "${pub_key_path} "]; then
```

Comment:
Is there a typo here: need an extra space between `"]` -> `" ]`

Reply:
ahh right, thanks for catching it